tests/test_basics.py

```diff
@@ -1,17 +1,17 @@
+import logging
 import os
 import unittest
-from unittest.mock import mock_open, patch, ANY, Mock
-import logging
+from typing import Mapping, Any
+
+from unittest.mock import ANY, Mock, mock_open, patch
 
 from loggo import Loggo as LoggoType
 
-test_setup = dict(facility='LOGGO_TEST',
-                  ip=None,
-                  port=None,
-                  do_print=True,
-                  do_write=True,
-                  private_data=['mnemonic', 'priv'])
-Loggo = LoggoType(test_setup)
+test_setup = dict(do_write=True,
+                  log_if_graylog_disabled=False,
+                  private_data={'mnemonic', 'priv'})  # type: Mapping[str, Any]
+
+Loggo = LoggoType(**test_setup)
```

Comment:
it would be nice to have the same style here as in the README. I.e. `Loggo` is the type and `loggo` is the instance

Reply:
sure. changed to lower in readme for pep but didn't do tests yet, can do